Twig/NewsTwigExtension.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of TheCadien/SuluNewsBundle.
+ *
+ * (c) Oliver Kossin
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace TheCadien\Bundle\SuluNewsBundle\Twig;
+
+use Doctrine\Common\Cache\Cache;
```

Comment:
Doctrine Cache is deprecated. See: https://www.doctrine-project.org/projects/doctrine-cache/en/2.2/index.html

You should consider moving to either a PSR-6 Cache or using Symfony's Cache contract.